contracts/recovery-contracts/RecoveryForwarder.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.18;
+
+/**
+ * Basic singleSig contract designed to send funds controlled by signer.
+ */
+contract RecoveryForwarder {
+  // Address which can move funds from this contract
+  address public signer;
+  /**
+   * Initialize the signer
+   */
+  function init(address _signer) external {
+    signer = _signer;
```

Comment:
need to check that its only initialized once... anyone can re-init to update the signer to whatever they want right now :D